tests/acceptance/standard/test_resource_groups.py

```diff
@@ -24,27 +24,27 @@ def add_gitlab_ci_config(group_and_project):
                 environment: production
                 resource_group: production
         """
-    run_gitlabform(add_gitlab_ci_config, group_and_project)
-    return group_and_project
+    run_gitlabform(add_gitlab_ci_config, project)
+    return project
 
 
 class TestResourceGroups:
     def test__update_resource_group_process_mode(
-        self, gitlab, group_and_project, add_gitlab_ci_config
+        self, gl, project, add_gitlab_ci_config
     ):
         update_resource_group_config = f"""
         projects_and_groups:
-          {group_and_project}:
+          {project.path_with_namespace}:
             resource_groups:
               production:
                 process_mode: newest_first
             """
 
         time.sleep(5)
-        run_gitlabform(update_resource_group_config, group_and_project)
+        run_gitlabform(update_resource_group_config, project)
 
-        resource_group = gitlab.get_specific_resource_group(
-            group_and_project, "production"
+        resource_group = gl.http_get(
+            f"/projects/{project.id}/resource_groups/production"
         )
         assert resource_group["key"] == "production"
         assert resource_group["process_mode"] == "newest_first"
```

Comment:
BTW @gdubicki this is a good example for the discussions we had in the issue last year. python-gitlab doesn't yet have support for `resource_groups` API, so here I can just use the generic `http_get()` method on the gitlab instance itself, and I still get dictionary access to attributes.

I've opened https://github.com/python-gitlab/python-gitlab/issues/2395 to finish it.

Reply:
No longer relevant, I bumped to 3.13.0 of python-gitlab and use the new `project.resources_groups.get()` endpoint.